guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -25,10 +14,12 @@ If you want unlimited hosts to register with the activation key, ensure the *Unl
 . From the *Environment* list, select the environment to use.
 . From the *Content View* list, select a content view to use.
 ifndef::orcharhino[]
-. In the *Repository Sets* tab, ensure only your named repository is enabled.
+. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired.
+For more information, see {ContentManagementDocURL}enabling-and-disabling-repositories-on-activation-key_content-management[Enabling and disabling repositories on activation key] in _{ContentManagementDocTitle}_.
 endif::[]
 . Click *Save*.
-. Optional: In the *System Purpose* section, you can configure the activation key with system purpose to set on hosts during registration.
+. Optional: In the *System Purpose* section, you can configure the activation key to set system purpose attributes on hosts during registration.
+This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

Comment:
```suggestion
This helps determine which repositories are available on the host.
It also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Or another way of splitting the long sentence.

Reply:
I don't see this anymore in my text editor.